### PR TITLE
Add doc for Dandelion

### PR DESCRIPTION
**Pull request by tduchateau**
Hi,

I've added docs for [Dandelion](http://dandelion.github.io/).

### CODE REVIEW

File: app/views/documentation.scala.html [hunk 1]
```diff
@@ -578,6 +579,58 @@ <h4 style="padding-top: 15px;">Public CDN via <a href="http://www.jsdelivr.com"
                 The right asset will be transparently accessed and served with proper HTTP caching behavior.
             </div>
 
+            <!-- Dandelion -->
+            <div class="tab-pane" id="wicket">
```
Comment (jamesward) [inline app/views/documentation.scala.html]: `id="dandelion"`

**Comment by tduchateau** [comment]
Oops. New commit added.

**Comment by tduchateau** [comment]
Thanks James!

**Comment by tduchateau** [comment]
Once again there's an error in the final link. I'll send a new PR ASAP. 
Sorry for the inconvenience :-/

**Pull request merged**